Tribler/Test/test_as_server.py

```diff
@@ -149,6 +153,16 @@ def tearDown(self, annotate=True):
             self.watchdog.print_all_stacks()
             raise RuntimeError("Couldn't stop the WatchDog")
 
+        if self.file_server:
+            deferred = self.file_server.stopListening()
```

Comment:
Quick note so you guys know: addCallback/errBack returns the deferred, so these three lines can be reduced to:

```
return self.file_server.stopListening().addCallback(self.checkReactor)
```